modules/ROOT/pages/functions/aggregating.adoc

```diff
@@ -1063,30 +1063,29 @@ The sum of the two supplied Durations is returned:
 [[grouping-keys]]
 == Grouping keys
 
-Aggregation expressions are expressions which contain one or more aggregation functions.
-A simple aggregation expression consists of a single aggregation function.
-For instance, `SUM(x.a)` is an aggregation expression that only consists of the aggregation function `SUM( )` with `x.a` as its argument.
-Aggregation expressions are also allowed to be more complex, where the result of one or more aggregation functions are input arguments to other expressions.
-For instance, `0.1 * (SUM(x.a) / COUNT(x.b))` is an aggregation expression that contains two aggregation functions, `SUM( )` with `x.a` as its argument and `COUNT( )` with `x.b` as its argument.
+Aggregating expressions are expressions which contain one or more aggregating functions.
+A simple aggregating expression consists of a single aggregating function.
+For instance, `SUM(x.a)` is an aggregating expression that only consists of the aggregating function `SUM( )` with `x.a` as its argument.
+Aggregating expressions are also allowed to be more complex, where the result of one or more aggregating functions are input arguments to other expressions.
+For instance, `0.1 * (SUM(x.a) / COUNT(x.b))` is an aggregating expression that contains two aggregating functions, `SUM( )` with `x.a` as its argument and `COUNT( )` with `x.b` as its argument.
```

Comment:
Not really related, but don't we usually write functions like `sum` and `count` with lowercase?